feature/src/main/java/com/teamdontbe/feature/notification/NotificationFragment.kt

```diff
@@ -74,50 +76,54 @@ class NotificationFragment :
+                    "beGhost" -> findNavController().navigate(R.id.action_notification_to_my_page)
+                    "contentGhost" -> navigateToHomeDetailFragment(notiData)
+                    "commentGhost" -> navigateToHomeDetailFragment(notiData)
+                    "userBan" -> Unit
+
+                    else ->
+                        Timber.tag("noti")
+                            .e("등록되지 않은 노티가 감지되었습니다 : ${notiData.notificationTriggerType}")
                 }
+            })
+        notiAdapter.apply {
+            pagingSubmitData(
+                viewLifecycleOwner,
+                notiViewModel.getNotificationList(),
+                notiAdapter,
+            )
+        }
 
-                is UiState.Empty -> Unit
-                is UiState.Failure -> Unit
-            }
-        }.launchIn(lifecycleScope)
-    }
+        if (binding.rvNotification.itemDecorationCount == 0) {
+            binding.rvNotification.addItemDecoration(
+                NotificationItemDecorator(requireContext()),
+            )
+        }
 
-    private fun initNotificationAdapter(notiData: List<NotiEntity>) {
-        if (notiData.isEmpty()) {
-            binding.layoutNotificationEmpty.visibility = View.VISIBLE
-        } else {
-            binding.rvNotification.adapter =
-                NotificationAdapter(click = { notiData, position ->
-                    when (notiData.notificationTriggerType) {
-                        "contentLiked" -> navigateToHomeDetailFragment(notiData)
-                        "comment" -> navigateToHomeDetailFragment(notiData)
-                        "commentLiked" -> navigateToHomeDetailFragment(notiData)
-                        "actingContinue" -> findNavController().navigate(R.id.action_notification_to_posting)
-                        "beGhost" -> findNavController().navigate(R.id.action_notification_to_my_page)
-                        "contentGhost" -> navigateToHomeDetailFragment(notiData)
-                        "commentGhost" -> navigateToHomeDetailFragment(notiData)
-                        "userBan" -> Unit
-
-                        else ->
-                            Timber.tag("noti")
-                                .e("등록되지 않은 노티가 감지되었습니다 : ${notiData.notificationTriggerType}")
-                    }
-                }).apply {
-                    submitList(notiData)
+        binding.rvNotification.adapter =
+            notiAdapter.withLoadStateFooter(footer = PagingLoadingAdapter())
+
+        notiAdapter.addLoadStateListener { combinedLoadStates ->
+            if (combinedLoadStates.append.endOfPaginationReached) {
+                // 아이템 수가 1보다 작으면 Empty View 보이도록 처리
+                if (notiAdapter.itemCount < 1) {
+                    binding.layoutNotificationEmpty.visibility = View.VISIBLE
+                } else {
+                    // 아이템 수가 1보다 크면 Empty View 안보이도록 처리
+                    binding.layoutNotificationEmpty.visibility = View.INVISIBLE
```

Comment:
p2 : 
```suggestion
                    binding.layoutNotificationEmpty.isVisible =if (notiAdapter.itemCount < 1)        
```
요렇게 고칠수도 있는데 isVisible false가 GONE상태라서 INVISIBLE이 꼭 필요한거면 안고쳐도 됩니다!

Reply:
넹.. 혹시나 해서 저렇게 짠거라 일단은 그냥 둘게욤